android_lib/src/main/java/io/sentry/godotplugin/SentryAndroidGodotPlugin.kt

```diff
@@ -0,0 +1,376 @@
+package io.sentry.godotplugin
+
+import android.util.Log
+import com.jakewharton.threetenabp.AndroidThreeTen
+import io.sentry.Attachment
+import io.sentry.Breadcrumb
+import io.sentry.Hint
+import io.sentry.Sentry
+import io.sentry.SentryEvent
+import io.sentry.SentryLevel
+import io.sentry.SentryOptions
+import io.sentry.android.core.SentryAndroid
+import io.sentry.protocol.Message
+import io.sentry.protocol.SentryException
+import io.sentry.protocol.SentryStackFrame
+import io.sentry.protocol.SentryStackTrace
+import io.sentry.protocol.User
+import org.godotengine.godot.Dictionary
+import org.godotengine.godot.Godot
+import org.godotengine.godot.plugin.GodotPlugin
+import org.godotengine.godot.plugin.UsedByGodot
+import org.godotengine.godot.variant.Callable
+import org.threeten.bp.format.DateTimeParseException
+import kotlin.random.Random
+
+
+@Suppress("unused")
+class SentryAndroidGodotPlugin(godot: Godot) : GodotPlugin(godot) {
+    companion object {
+        private const val TAG = "sentry-godot"
+    }
+
+    private val eventsByHandle = ThreadLocal.withInitial { mutableMapOf<Int, SentryEvent>() }
+    private val exceptionsByHandle = ThreadLocal.withInitial { mutableMapOf<Int, SentryException>() }
+
+    private fun getEvent(eventHandle: Int): SentryEvent? {
+        val event: SentryEvent? = eventsByHandle.get()!![eventHandle]
```

Comment:
~If `eventsByHandle.get()` is null, it'll throw because of the `!!`, but we have a null check and a log below. So the indexer syntax can return null?
should we replace `!!` with `?.` instead?~
```suggestion
        val event: SentryEvent? = eventsByHandle.get()?.[eventHandle]
```

why. do we need `!!` to access `eventsByHandle` if we instantiate it with `withInitial`?